ISICkonto/Support/UI/FormField.swift

```diff
@@ -43,4 +43,9 @@ class FormField: UITextField, UITextFieldDelegate{
     override func textRect(forBounds bounds: CGRect) -> CGRect {
         return bounds.insetBy(dx: 10, dy: 0)
     }
+    
+    public func deactivate() {
```

Comment:
tohle nikde nevoláš, nebo jo? 🤔 jakým způsobem používáš `public`, `private`, atd? Někde to je napsané, někde ne a je to takové zajímavé použití 😊

Reply:
mela jsem to nachystane, protoze to chci volat
co se tyce public private atd, tak to jeste nemam poradne zmakle, ale muj dalsi krok je si to nacist